src/chigame/games/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+from chigame.users.models import FriendGroup, User
+
+
+class Game(models.Model):
+    """
+    A game (Chess, Checkers, etc.)
+    """
+
+    name = models.TextField()
+    description = models.TextField()
+    min_players = models.PositiveIntegerField()
+    max_players = models.PositiveIntegerField()
```

Comment:
Do we want to provide options for a game type of any kind? (turn based, draft, real time, etc...)
Or leave that to the games team.

Reply:
I don't think we want to get ahead of ourselves and start modeling that kind of stuff when we don't yet know what kind of games we're going to end up supporting. The games team could add fields to that effect, but I think it should be driven by the actual games we end up supporting.